fe/fe-core/src/test/java/com/starrocks/analysis/SelectStmtTest.java

```diff
@@ -343,4 +343,27 @@ public void testScalarCorrelatedSubquery() {
             Assert.fail("Should not throw an exception");
         }
     }
+
+    @Test
+    public void testMultiDistinctMultiColumnWithLimit() throws Exception {
+        String[] sqlList = {
+                "select count(distinct k1, k2), count(distinct k3) from db1.tbl1 limit 1",
```

Comment:
could we directly remove the limit 1 for these sqls? because the limit is meaningless. then, we needn't change the MergeLimitDirectRule

Reply:
both non-group-by and  group-by agg query has these problem,  limit-clause in group-by agg is not eliminated, so we should change MergeLimitDirectRule; while in non-group-by agg, limit-clause can be eliminated, but this is not the intention of this PR,  another PR can do these.